### ISSUE TITLE
PHP SSL error on one particular website

### ISSUE THREAD

**Issue by LoveMyData** [openaustralia/morph#951]
I have two scrapers LoveMyData/port_macquarie_hastings and LoveMyData/wollondilly

port_macquarie_hastings is working fine and it is using HTTPS

wollondilly is not working and throwing error messages like 
PHP Warning: file_get_contents(): SSL operation failed with code 1.

It is working locally and I have disabled SSL check in PHP as well as force to use TLS v1 or v2, no different....

I think the same issue are showing on other languages
https://github.com/openaustralia/morph/issues/940 - Perl
https://github.com/openaustralia/morph/issues/939 - Ruby

I think we just need to patch the php version to the latest build within the same branch and it should resolve it as I am using the latest branch release on mac with no issue.

**Comment by henare** [comment]
>I think we just need to patch the php version to the latest build within the same branch and it should resolve it as I am using the latest branch release on mac with no issue.

Have you tried [choosing a different version](https://morph.io/documentation/language_version) of PHP on morph.io to see if it does fix the problem?

**Comment by LoveMyData** [comment]
@henare , tried v5.5.21and v5.6.5 which are the latest available on morph platform and still not working. :-(

Latest are v5.5.30 and v5.6.16

**Comment by adricl** [comment]
@henare Is there an easy way to clone this project with the same MiM proxy setup so we can test this locally?

**Comment by henare** [comment]
@adricl yep, just [uncomment the line](https://github.com/openaustralia/morph/blob/master/Procfile#L11) in the Procfile and mitmproxy will run when you `foreman start` :+1: (I think that's all you need to do anyway).

It would be great to get this fixed - thanks for looking into it! :four_leaf_clover:

**Comment by benrfairless** [comment]
Closing as stale - this is one of several 2015-2019 reports of HTTPS failures caused by the mitmproxy interception layer. The specific scrapers and sites involved are long gone, and openaustralia/oaf-internal#308 proposes removing the separate proxy entirely. If HTTPS failures persist on the current platforms, please open a fresh issue with a current example.

This was carried out with Claude Code 2.1.198 (Claude Fable 5) after being reviewed by me.
